.github/workflows/docbuild.yml

```diff
@@ -58,39 +58,9 @@ jobs:
         run: |
           export PATH="$HOME/.local/bin:$PATH"
           mkdir -p _build && cd _build
-          cmake -GNinja ../nrf/doc
+          cmake -GNinja ../nrf/doc -DEXTRA_SPHINXOPTS="-W"
           ninja build-all
 
-      - name: Check build warnings
```

Comment:
Where are you checking if there are warnings now? Just the ninja exit code?

Reply:
Return code will be non-zero:
```
Warning, treated as error:
/.../ncs/nrf/doc/_build/nrf/rst/doc/nrf/include/bluetooth/mesh/light_ctl_cli.rst:35:Duplicate C declaration, also defined at include/bluetooth/mesh/light_ctl:35.
Declaration is '.. c:function:: int bt_mesh_light_ctl_set (struct bt_mesh_light_ctl_cli *cli, struct bt_mesh_msg_ctx *ctx, const struct bt_mesh_light_ctl_set *set, struct bt_mesh_light_ctl_status *rsp)'.
ninja: build stopped: subcommand failed.
ninja: build stopped: subcommand failed.
(.venv) ➜  nrf git:(feature/remove-known-issues) ✗ echo $?
1
```

Reply:
So if we have an error in the Zephyr build (locally), we cannot use build-all?

And we don't have any files anymore that contain the errors?

Reply:
Yes, we can. `-W` option is only enforced on CI.

Reply:
regarding errors, see the `doc/nrf/known-warnings.txt` and the updated docs detailing the mechanism.